inlang/source-code/versioned-interfaces/message-bundle/package.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "@inlang/message-bundle",
+	"type": "module",
+	"version": "0.0.1",
```

Comment:
Do interfaces need to have a major version >0

Reply:
No, a `0.x.x` version is fine